govcd/tm_ldap.go

```diff
@@ -218,13 +217,11 @@ func ldapExecuteRequest(vcdClient *VCDClient, orgId, method string, payload inte
 	// If the call is PUT, we prepare the body with the input settings
 	var body io.Reader
 	if method == http.MethodPut {
-		text := bytes.Buffer{}
-		encoder := json.NewEncoder(&text)
-		err = encoder.Encode(payload)
+		text, err := json.Marshal(payload)
```

Comment:
This was an attempt to make it work as I wanted. Didn't make it, but the code is simpler so I left it